docs/tempo/website/configuration/manifest.md

```diff
@@ -20,6 +20,7 @@ go run ./cmd/tempo --storage.trace.backend=local --storage.trace.local.path=/tmp
 
```

Comment:
I can't put a review comment on the actual line, but can you update the date on the line above?

> **Note**: This manifest was generated on 2021-10-27.

Thanks for including the new metrics-generator changes 🙏🏻